src/main/java/org/isf/lab/rest/LaboratoryController.java

```diff
@@ -134,8 +177,9 @@ public ResponseEntity<Boolean> newLaboratory(@RequestBody LabWithRowsDTO labWith
 	}
 
 	@PostMapping(value = "/laboratories/insertList", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<Boolean> newLaboratory2(@RequestBody List<LabWithRowsDTO> labsWithRows) throws OHServiceException {
```

Comment:
Why is this API camel cased (`insertList`) where the one in line 138 above isn't (`examrequest`)?    Let's be consistent.

Reply:
I agree... @ArnaudFonzam can you do it ASAP? I've already merged CORE and GUI sides... Thanks!

Reply:
Please check the whole file for this issue; thanks